{
    "identifier": "VULCODE-2020-09-26 12:48:47.337840",
    "summary": "Lander 0.1.15 updates 'requests' to 2.20.0 for security reasons.",
    "references": [
        {
            "url": "",
            "reference_id": "pyup.io-38375"
        }
    ],
    "vulnerable_packages": [
        "pkg:pypi/lander@0.1.8",
        "pkg:pypi/lander@0.1.7",
        "pkg:pypi/lander@0.1.0",
        "pkg:pypi/lander@0.1.6",
        "pkg:pypi/lander@0.1.5",
        "pkg:pypi/lander@0.1.0b1",
        "pkg:pypi/lander@0.1.1",
        "pkg:pypi/lander@0.1.14",
        "pkg:pypi/lander@0.1.0b5",
        "pkg:pypi/lander@0.1.3",
        "pkg:pypi/lander@0.1.14a2",
        "pkg:pypi/lander@0.1.14a1",
        "pkg:pypi/lander@0.1.11",
        "pkg:pypi/lander@0.1.0b2",
        "pkg:pypi/lander@0.1.9",
        "pkg:pypi/lander@0.1.13",
        "pkg:pypi/lander@0.1.0b4",
        "pkg:pypi/lander@0.1.0b3",
        "pkg:pypi/lander@0.1.14a4",
        "pkg:pypi/lander@0.1.0b7",
        "pkg:pypi/lander@0.1.0b8",
        "pkg:pypi/lander@0.1.12",
        "pkg:pypi/lander@0.1.0b6",
        "pkg:pypi/lander@0.1.14a3",
        "pkg:pypi/lander@0.1.0a1",
        "pkg:pypi/lander@0.1.2",
        "pkg:pypi/lander@0.1.4",
        "pkg:pypi/lander@0.1.10"
    ],
    "resolved_packages": [
        "pkg:pypi/lander@0.1.15",
        "pkg:pypi/lander@0.2.0b8",
        "pkg:pypi/lander@0.1.16",
        "pkg:pypi/lander@0.2.0b5",
        "pkg:pypi/lander@0.2.0b7",
        "pkg:pypi/lander@0.2.0b6",
        "pkg:pypi/lander@0.2.0"
    ]
}